data/wicked/arp-tool.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+import scapy.all as scapy
+import ipaddress
+import time
+import argparse
+import sys
+
+
+def defend(interface, network, count):
+    myMAC = scapy.get_if_hwaddr(interface)
+    counter = 0
+    ipaddress.ip_network(network)
```

Comment:
```suggestion
```

We could make `argparse` check for a valid network with this function and the suggestion shown later.

```
def network(string):
    return ipaddress.ip_network(string)
```

This way, the script would fail with:
`arp-tool.py: error: argument --network: invalid network value: 'xxx'
`

Reply:
wow nice, thx!